mempool/clist_mempool.go

```diff
@@ -410,36 +397,29 @@ func (mem *CListMempool) resCbFirstTime(
 				return
 			}
 
-			memTx := &mempoolTx{
+			mem.addTx(&mempoolTx{
 				height:    mem.height,
 				gasWanted: r.CheckTx.GasWanted,
 				tx:        tx,
-			}
-			memTx.addSender(txInfo.SenderID)
-			mem.addTx(memTx)
+			})
 			mem.logger.Debug(
-				"added good transaction",
+				"added valid transaction",
```

Comment:
+1

in general, it would be nice to use valid/invalid 
instead of their synonyms everywhere